spec/controllers/changes_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
         @destroy_version = @destroyed.versions.last
       end
     end
-    let(:submit) { get :undo, id: version.id }
+    let(:submit) { get :undo, params: { id: version.id } }
```

Comment:
It's fine though, there's not a better way, I just don't like how it looks personally.